docs/public-networks/how-to/upgrade-node.md

```diff
@@ -6,28 +6,171 @@ tags:
   - public networks
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Upgrade your Besu node
 
 When upgrading your Besu node, we recommend:
 
-- Using an orchestration method (for example, Ansible or Chef) to keep all nodes in sync with your desired configuration.
+- Preserving your node's data and configuration
+<!--- - Using an orchestration method  to keep all nodes in sync with your desired configuration. --->  
 - Storing your configuration under version control.
 
-## Ansible
 
-You can use the [Ansible role on Galaxy](https://galaxy.ansible.com/consensys/hyperledger_besu) directly or customize it to suit your needs.
+<Tabs>
+  <TabItem value="binary" label="Linux" default>
+
+### Upgrading on Linux
+
+1. Stop your Besu node:
+
+```bash
+sudo systemctl stop besu
+```
+
+2. Back up your data directory and configuration:
+
+```bash
+sudo cp -r /var/lib/besu/data /var/lib/besu/data.backup
```

Comment:
For mainnet, the data directory is huge and most stakers wouldn't be able to back it up without running out of space and potentially breaking their node.

Reply:
The link in my issue was broken, it should have been https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet/part-ii-maintenance/updating-execution-client#besu